docs/reference/streamfield/blocks.md

```diff
@@ -0,0 +1,462 @@
+(streamfield_block_reference) =
+
+# StreamField block reference
+---
+This document details the block types provided by Wagtail for use in [StreamField](../../topics/streamfield.rst), and how they can be combined into new block types.
+
+***class* wagtail.fields.StreamField(blocks, use_json_field=None, blank=False, min_num=None, max_num=None, block_counts=None, collapsed=False)**
```

Comment:
Something has gone wrong with this conversion, maybe look at other similar markdown files OR use the eval to just eval the original RST here.

e.g.

```
   ```{eval-rst}
       .. class:: wagtail.embeds.models.Embed
```

It looks like you have reproduced the autodoc content with markdown, however this is meant to use the documentation system's approach.

Maybe use `docs/reference/pages/panels.md` or `docs/advanced_topics/embeds.md` as a reference of how this is done.

While this is not autodoc you can see a similar feature documented here https://docs.wagtail.org/en/latest/contributing/documentation_guidelines.html#autodoc